src/renderer/components/item-object-list/content.tsx

```diff
@@ -68,67 +68,76 @@ export class ItemListLayoutContent<I extends ItemObject> extends React.Component
     return this.props.store.failedLoading;
   }
 
+  @boundMethod
+  renderRow(item: I) {
+    return this.getTableRow(item);
+  }
+
+  getTableRow(item: I) {
```

Comment:
this function implementation is cut and paste from getRow() below, no functional change